ros_msft_onnx/CMakeLists.txt

```diff
@@ -25,47 +34,76 @@ find_package(catkin REQUIRED COMPONENTS
   tf
 )
 
-get_cmake_property(_variableNames VARIABLES)
-foreach (_variableName ${_variableNames})
-  #message(STATUS "${_variableName}=${${_variableName}}")
-endforeach()
-
-
 catkin_package(
   INCLUDE_DIRS include
-  CATKIN_DEPENDS std_msgs geometry_msgs visualization_msgs ros_msft_onnx_msgs image_transport roscpp cv_bridge tf onnxruntime_vendor
+  CATKIN_DEPENDS std_msgs geometry_msgs visualization_msgs ros_msft_onnx_msgs image_transport roscpp cv_bridge tf
+  CFG_EXTRAS  "onnxruntime_vendor-extras.cmake"
 )
 
 include_directories(
   include
   ${catkin_INCLUDE_DIRS}
   ${EIGEN3_INCLUDE_DIRS}
-  ${onnxruntime_vendor_INCLUDE_DIRS}
 )
 
 add_executable(${PROJECT_NAME}_node src/ros_msft_onnx.cpp src/main.cpp src/yolo_box.cpp src/pose_parser.cpp)
-add_dependencies(${PROJECT_NAME}_node ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS} ${onnxruntime_vendor_TARGETS})
-target_link_libraries(${PROJECT_NAME}_node ${catkin_LIBRARIES} ${OpenCV_LIBRARIES} ${onnxruntime_vendor_TARGETS})
+add_dependencies(${PROJECT_NAME}_node ${catkin_EXPORTED_TARGETS})
+target_link_libraries(${PROJECT_NAME}_node ${catkin_LIBRARIES} ${OpenCV_LIBRARIES} ${EIGEN3_LIBS})
+
+message("Installing onnxruntime_vendor Nuget package")
+
+if(CUDA_SUPPORT)
```

Comment:
It looks like a duplicate to what we did in `onnxruntime_vendor`?

Reply:
(Ah I saw you deleted it, curious why?)

Reply:
I am not sure what you exactly mean here

Reply:
Maybe you need to rebase your branch with the latest master. From the GitHub pull request, it says you deleted a common package called `onnxruntime_vendor`.

Reply:
When I try to build this on Windows I get the following error, could be related?:

```
CMake Error in ros_msft_onnx/ros_msft_onnx/CMakeLists.txt:
  Imported target "onnxruntime-unofficial::onnxruntime" includes non-existent
  path

    "C:/Onnx_ML/onnx_custom_vision/install/include/onnxruntime_vendor"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.
```